qa/sys/debian/user_bootstrap.sh

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env bash
 
 VERSION=`cat /vagrant/config/platforms.json | grep  latest | cut -d":" -f2 | sed 's/["\|,| ]//g'`
-LOGSTASH_FILENAME="logstash-${VERSION}_all.deb"
+LOGSTASH_FILENAME="logstash-${VERSION}.deb"
```

Comment:
why is this change in this PR? I'm not familiar with the naming requirements, are we removing architectures from package filenames?